docs/user-guide/banner-section.md

```diff
@@ -0,0 +1,20 @@
+# Banner Section
+**********************
+
+The Banner Section is a [Title Section](title-section.md) without the text content to create a banner in the story through the background content.
+
+<img src="../img/banner-section/banner-section.jpg" class="ms-docimage"/>
+
+By the background editing toolbar the user can:
+
+* Add a media content as a background opening the [Media Editor](media-editor-window.md) through the **Change media source** button <img src="../img/button/change-media.jpg" class="ms-docbutton"/> and choose between images, videos or maps. 
+
+* Change the height of the section through the **Fit/adapt content** button <img src="../img/button/fit-adapt-content.jpg" class="ms-docbutton"/>
+
+* Remove the banner section through the **Remove** button <img src="../img/button/remove-button-banner.jpg" class="ms-docbutton"/>
+
+Once an image, a video or a map is added to the background the editing toolbar change to enabling different functionality depending on the content inserted: as is explained [here](title-section.md#images).
```

Comment:
Once the media content has been added as a background of the section, the editing toolbar changes to enable different functionality depending on the content inserted: as explained  [here](title-section.md#images).